integration-tests/tests/src/node/path.ts

```diff
@@ -57,6 +59,32 @@ describe("path configuration (local)", function () {
   });
 });
 
+describe.skipIf(environment.missingServer, `app configuration of root directory (flexible sync)`, async function () {
+  // describe.only(`app configuration of root directory (flexible sync)`, async function () {
+  this.timeout(60_000);
+  const tmpdir = getAbsolutePath();
+  importAppBefore(buildAppConfig("with-flx").baseFilePath(tmpdir).anonAuth().flexibleSync());
```

Comment:
Let's use the `sdkConfig` param of `importAppBefore` to set the properties of the local `Realm.App`instance.  The builder parts should just be used for configuring the remote BaaS instance.
```suggestion
  importAppBefore(buildAppConfig("with-flx").anonAuth().flexibleSync(), {baseFilePath:tmpDir});
```

Reply:
The commit suggestion will not as `App.Configuration` (second argument) will require `id`. `importAppBefore()` could take a relaxed version of `App.Configuration` (only optional parameters)